content/tbb/how-to-verify-signature/contents.lr

```diff
@@ -40,6 +40,12 @@ If you are using GNU/Linux, then you probably already have GnuPG in your system,
 
 In order to verify the signature you will need to type a few commands in a terminal window.  How to do this will vary depending on your distribution.
 
+### For Android users:
+
+If you are using Android, you need to install the app Checkey to show the signature of your application. It can be installed from [F-droid](https://f-droid.org/en/packages/info.guardianproject.checkey). 
+
+APK files are signed java JAR files. In order to verify the signature, you will need to type commands in either the application [jarsigner](https://docs.oracle.com/javase/9/tools/jarsigner.htm) or [apksigner](https://developer.android.com/studio/command-line/apksigner).
```

Comment:
I feel it's a bit unclear if this step can be done using the phone only or needs a computer, maybe the instructions could be detailed a bit? Also, I'm not super sure but can't just the signature that is shown in Checkey be compared to the one on the website?

https://trac.torproject.org/projects/tor/ticket/27514 also has the relevant commands that should be ported over for uniformity maybe.